Extensions/TextInput/textinputruntimeobject-pixi-renderer.ts

```diff
@@ -246,8 +271,9 @@ namespace gdjs {
     }
 
     updateOpacity() {
-      if (!this._input) return;
-      this._input.style.opacity = '' + this._object.getOpacity() / 255;
+      if (!this._form) return;
+      this._form.style.opacity =
+        '' + (this._object.getOpacity() / 255).toFixed(3);
```

Comment:
```suggestion
        (this._object.getOpacity() / 255).toFixed(3);
```
toFixed returns as string already